MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -61,6 +62,7 @@ It is also important to note that while this document provides the criteria agai
   * -_note: these tests are defined in https://github.com/folio-org/folio-integration-tests_
 * Data is segregated by tenant at the storage layer (6, 7)
 * The module doesn’t access data in DB schemas other than its own and public (6, 7)
+* Any dependencies, other than on defined interfaces, are declared in the README.md.
```

Comment:
@jeremythuff This is what @craigmcnally @starsplatter and I came up with to address your concern.